types/morphdom/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for morphdom 2.3
+// Project: https://github.com/patrick-steele-idem/morphdom
+// Definitions by: arvitaly <https://github.com/arvitaly>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface MorphDomOptions {
+    getNodeKey?: (node: Node) => any;
+    onBeforeNodeAdded?: (node: Node) => Node;
+    onNodeAdded?: (node: Node) => Node;
+    onBeforeElUpdated?: (fromEl: HTMLElement, toEl: HTMLElement) => boolean;
+    onElUpdated?: (el: HTMLElement) => void;
+    onBeforeNodeDiscarded?: (node: Node) => boolean;
+    onNodeDiscarded?: (node: Node) => void;
+    onBeforeElChildrenUpdated?: (fromEl: HTMLElement, toEl: HTMLElement) => boolean;
+    childrenOnly?: boolean;
+}
+
+declare namespace morphdom { }
```

Comment:
Can you please remove the empty namespace?

Reply:
To elaborate, this is an inappropriate hack. [Please read more here from the DefinitelyTyped REAMDE](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)